### PR TITLE
Cv 714 for Cohort review summaries

### PR DESCRIPTION
**Pull request by Pauljgraham**
This comparison is based on CV-713 (needs to be merged into master!!!!! Please retarget when merging

### CODE REVIEW

File: src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Mappers/Cohort/ReviewRequestViewModelMapper.cs [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        public async Task<ReviewViewModel> Map(CohortsByProviderRequest source)
+        {
+            var cohortsResponse = await _commitmentsApiClient.GetCohorts(new GetCohortsRequest { ProviderId = source.ProviderId });
+
+            var reviewViewModel = new ReviewViewModel
+            {
+                ProviderId = source.ProviderId,
+                Cohorts = cohortsResponse.Cohorts
+                    .Where(x => x.GetStatus() == CohortStatus.Review)
+                    .OrderByDescending(z => z.CreatedOn)
```
Comment (hkf-tech) [inline src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Mappers/Cohort/ReviewRequestViewModelMapper.cs:33]: The order by is Ascending on employer.

File: src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web.UnitTests/Controllers/CohortControllerTests/WhenGettingCohorts.cs [hunk 2]
```diff
@@ -17,49 +17,72 @@ public class WhenGettingCohorts
...
+        public async Task ThenReturnsCohortsViewModel()
         {
-            var fixture = new WhenGettingCohortsFixture();
+            var f = new WhenGettingCohortsFixture();
 
-            var result = await fixture.Act() as ViewResult;
+            var result = await f.Sut.Cohorts(f.Request) as ViewResult;
 
             Assert.NotNull(result);
             Assert.AreEqual(typeof(CohortsViewModel), result.Model.GetType());
```
Comment (hkf-tech) [inline src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web.UnitTests/Controllers/CohortControllerTests/WhenGettingCohorts.cs:35]: Probably the extension method is more readable, but not fussed about it. Leave it to you if you want to change.
result.
VerifyReturnsViewModel().WithModel<CohortsViewModel>();

File: src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Mappers/Cohort/ReviewRequestViewModelMapper.cs (hunk 1 above)
Comment (Pauljgraham) [reply src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Mappers/Cohort/ReviewRequestViewModelMapper.cs]: added

File: src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Views/Cohort/Review.cshtml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+                <tbody class="govuk-table__body">
+                    @foreach (var cohort in Model.Cohorts)
+                    {
+                        <tr class="govuk-table__row" data-cohort="@cohort.CohortReference">
+                            <td class="govuk-table__cell" data-label="Training provider">@cohort.EmployerName</td>
+                            <td class="govuk-table__cell" data-label="Reference">@cohort.CohortReference</td>
+                            <td class="govuk-table__cell govuk-table__cell--numeric" data-label="Number of apprentices">@cohort.NumberOfApprentices</td>
+                            <td class="govuk-table__cell" data-label="Last Message">@cohort.LastMessage</td>
+                            <td class="govuk-table__cell govuk-table__cell--numeric" data-label="">
+                                <a class="govuk-link cohort-details-link" href="@Url.ProviderApprenticeshipServiceLink($"{Model.ProviderId}/apprentices/{cohort.CohortReference}/details")">Details</a>
```
[the remark this answers is not in the document]
Comment (Pauljgraham) [reply src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Views/Cohort/Review.cshtml]: added

**Pull request closed without merging**